docker/pypi/wmagent-mariadb/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+manage init-mariadb 2>&1 | tee -a run.log
+manage start-mariadb 2>&1 | tee -a run.log
+
+echo "Start sleeping....zzz"
+while true; do sleep 10; done
```

Comment:
Hi @vkuznet you are absolutely correct, and this is exactly why I did not redirect the `stdout`, but rather forked and duplicated it with `tee`. So it is still preserved in docker logs:
```
cmst1@vocms0290:wmagent-mariadb $ docker ps 
CONTAINER ID   IMAGE                COMMAND      CREATED        STATUS        PORTS     NAMES
658163bd2eaa   local/mariadb:10.5   "./run.sh"   12 hours ago   Up 12 hours             mariadb
cmst1@vocms0290:wmagent-mariadb $ docker  logs mariadb
-------------------------------------------------------------------------
Stopping any previously running mariadb server
mariadb-admin: connect to server at 'localhost' failed
error: 'Can't connect to local MySQL server through socket '/data/srv/mariadb/10.5/mariadb.sock' (2)'
Check that mysqld is running and that the socket: '/data/srv/mariadb/10.5/mariadb.sock' exists!

-------------------------------------------------------------------------
Trying to install system database if it is not present already
mysql.user table already exists! Run mysql_upgrade, not mysql_install_db
WARNING: System and user databases already exist. NOT trying to create them.
starting MariaDB server
...
240226 18:24:13 mysqld_safe Logging to '/data/srv/mariadb/10.5/logs/error.log'.
240226 18:24:13 mysqld_safe Starting mariadbd daemon with databases from /data/srv/mariadb/10.5/install/database
mariadb-admin  Ver 9.1 Distrib 10.5.24-MariaDB, for debian-linux-gnu on x86_64
Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Server version		10.5.24-MariaDB-1:10.5.24+maria~ubu2004-log
Protocol version	10
Connection		Localhost via UNIX socket
UNIX socket		/data/srv/mariadb/10.5/mariadb.sock
Uptime:			10 sec

Threads: 1  Questions: 1  Slow queries: 0  Opens: 16  Open tables: 10  Queries per second avg: 0.100

Uptime: 10  Threads: 1  Questions: 2  Slow queries: 0  Opens: 16  Open tables: 10  Queries per second avg: 0.200

Start sleeping....zzz
```
The content is the same as the contents of the `run.log` from inside of the container. And since, this log I consider useful only for monitoring the current container startup process, I did not bothered to over complicate the situation and redirect it to a mounted point for long term preservation.